python/lsst/ts/atwhitelight/atwhitelight_csc.py

```diff
@@ -390,6 +396,12 @@ async def do_turnLampOn(self, data):
             power = self.config.lamp.default_power
         else:
             power = data.power
+        await self.cmd_turnLampOn.ack_in_progress(
+            data=data,
+            timeout=self.config.lamp.max_lamp_on_delay + ONOFF_COMMAND_TIMEOUT_MARGIN,
+        )
+        # Note: ``reason`` is used if an existing turnLampOff command
+        # has to be aborted; hence the use of "Superseded".
```

Comment:
Does this comment belong here? No use of `Superseded` in the next line of code.

Reply:
Good catch! I had a reason argument for LampModel.turn_lamp_on for awhile, but realized it was not needed. When I removed it I forgot to remove this comment.

The only reason to turn the lamp on is if a user commands it (whereas the lamp may be turned off for that reason or because of going to fault).